src/graph/InsertEdgeExecutor.cpp

```diff
@@ -125,6 +137,78 @@ StatusOr<std::vector<storage::cpp2::Edge>> InsertEdgeExecutor::prepareEdges() {
             values.emplace_back(ovalue.value());
         }
 
+        if (expressions.size() != schema_->getNumFields()) {
```

Comment:
this function is too long, maybe should separate it to some sub function?

Reply:
I have extracted some common parts.